knimin/lib/data_access.py

```diff
@@ -3191,6 +3186,252 @@ def delete_sample_plate(self, id):
             TRN.add(sql, [id])
             TRN.execute()
 
+    def get_id_by_name(self, field, name):
```

Comment:
Also, it seems that none of this functions are tested. I recommend to do another PR in which only this functions are added and reviewed. It is really hard to review this...